cluster-provision/k8s/1.31/manifests/prometheus/grafana/grafana-deployment.yaml

```diff
@@ -201,9 +243,9 @@ spec:
       - configMap:
           name: grafana-dashboard-scheduler
         name: grafana-dashboard-scheduler
-      - configMap:
-          name: grafana-dashboard-statefulset
-        name: grafana-dashboard-statefulset
       - configMap:
           name: grafana-dashboard-workload-total
         name: grafana-dashboard-workload-total
+      - name: grafana-config
```

Comment:
it looks like you are missing this secret 

```
./cluster-up/kubectl.sh describe pod -n monitoring grafana-7f998f499d-cs58h | grep Warning
selecting podman as container runtime
  Warning  FailedMount  79s (x9 over 3m27s)  kubelet            MountVolume.SetUp failed for volume "grafana-config" : secret "grafana-config" not found
```

Reply:
@brianmcarey  Thankyou